index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Ajax Reddit Slideshow</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
```

Comment:
Here, I would definitely add a title and description of how your program works, so that users know how to use the site.